src/internal_events/http.rs

```diff
@@ -0,0 +1,54 @@
+use super::InternalEvent;
+use metrics::counter;
+
+#[derive(Debug)]
+pub struct HTTPEventsReceived {
+    pub events_count: usize,
+    pub byte_size: usize,
+}
+
+impl InternalEvent for HTTPEventsReceived {
+    fn emit_logs(&self) {
+        debug!(
```

Comment:
I'm not completely sure, but I think this should be `trace!`. #3285

Reply:
Yeah, this one is a little different given that it's processing a batch of events. @juchiast let's move it to `trace!` for now.

Reply:
Changed to trace!